agent/security_helper.py

```diff
@@ -83,38 +83,45 @@ def is_app_armor_enabled():
     Returns a True/False if AppArmor is enabled.
     """
     try:
-        from sh import aa_status
+        import LibAppArmor
+        return LibAppArmor.aa_is_enabled() == 1
     except ImportError:
-        return False
+        try:
+            from sh import aa_status
+        except ImportError:
+            return False
 
-    # Returns 0 if enabled and 1 if disable
-    get_aa_status = aa_status(['--enabled'], _ok_code=[0, 1]).exit_code
-    if get_aa_status == 1:
-        return False
-    return True
+        # Returns 0 if enabled and 1 if disabled
+        return aa_status(['--enabled'], _ok_code=[0, 1]).exit_code != 1
```

Comment:
it's not clear why didn't you put **all** possible return codes of `aa_status` to `_ok_code`

Reply:
fair enough, needs fixing